buildSrc/src/main/kotlin/io/getstream/chat/android/Dependencies.kt

```diff
@@ -175,6 +175,7 @@ object Dependencies {
     const val mockitoKotlin = "com.nhaarman.mockitokotlin2:mockito-kotlin:${Versions.MOCKITO_KOTLIN}"
     const val moshi = "com.squareup.moshi:moshi:${Versions.MOSHI}"
     const val moshiCodegen = "com.squareup.moshi:moshi-kotlin-codegen:${Versions.MOSHI}"
+    const val moshiKotlin =  "com.squareup.moshi:moshi-kotlin:${Versions.MOSHI}"
```

Comment:
I didn't want to mark our models with `@JsonClass(generateAdapter = true)` that's why I need an adapter for Kotlin